doc/configuration.txt

```diff
@@ -2958,6 +2959,9 @@ maxzlibmem <number>
   with "show info" on the line "MaxZlibMemUsage", the memory used by zlib is
   "ZlibMemUsage" in bytes.
 
+no-mptcp
```

Comment:
same here: should be moved under `no-memory-trimming` if it is by alphabetical order